docs/03_CONFIGURATION.md

```diff
@@ -93,6 +29,7 @@ There are also environment secrets for each environment (`staging`, `staging-alt
 | `DEPLOY_ENV`                | Deployment environment on elastic beanstalk.                                      |
 | `REACT_APP_FORMSG_SDK_MODE` | Determines the keys used in the formsg SDK. Set either `production` or `staging`. |
 
+
 ## Environment Variables
 
```

Comment:
Can remove this section regarding EB configs.

```The environment for each group is user-defined, and should be specified in the Elastic Beanstalk configuration
as the environment variable `SSM_PREFIX`.
The specific environment is user-defined, and should be specified in the Elastic Beanstalk configuration
as the environment variable `SSM_ENV_SITE_NAME`. This variable is optional.
The list of categories can be inferred by looking at the file `.ebextensions/env-file-creation.config`.```

Also can remove mention of SSM_PREFIX which is specific to EB.
```
| `SSM_PREFIX` | String prefix (typically the environment type) for AWS SSM parameter names to create a .env file for FormSG. (`staging`, `prod`, `uat`) |
```